.envrc

```diff
@@ -0,0 +1,27 @@
+# This file is used by direnv to setup the environment when entering to use fvm's flutter.
+
+# Comment out the next line if you want to use your system flutter.
+PATH_add .fvm/flutter_sdk/bin
+
+# Check flutter version matches what's in .fvmrc
+check_flutter_version() {
```

Comment:
This file is a shell script, so let's include it in `tools/check shellcheck`. That means in the `run_shellcheck` function, adding this file to both the `files_check` line and the `targets` array.

(Then that will give a few warnings on this version, which you can fix.)